www/.env.development

```diff
@@ -1,2 +1,2 @@
 SEGMENT_KEY=''
-FEATURE_ENABLE_AXE=false
+FEATURE_ENABLE_AXE=true
```

Comment:
nit: Maybe wrap `true` in single quotes to make it explicitly known as a string at first glance.